src/backend/src/utils/slack.utils.ts

```diff
@@ -1,4 +1,4 @@
-import { ChangeRequest, daysBetween, Task, User, wbsPipe, WorkPackage } from 'shared';
+import { ChangeRequest, daysBetween, ReimbursementRequest, Task, User, wbsPipe, WorkPackage } from 'shared';
```

Comment:
so this is a linting error, it's telling you, you should delete the ReimbursementRequest import since it's not used